### PR TITLE
Completing the challenge

### PR DESCRIPTION
**Pull request by dikshagoyal26**
Hi Mentor!
I have completed All 5 projects and 3 Blogs, please review and accept my pull request.

I know I have done a really silly mistake in the pull request #1094 and I regret it. But the projects I have made are completely mine. I haven't copied anything from any other participant. So, I would like to request you to please reconsider the candidature.

Thank you.

### CODE REVIEW

File: content/finalist/ttjc@finalist@dikshagoyal26/index.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+### Tic Tac Toe
+
+A react implementation of the Tic Tac Toe game.
+
+- **Stack** `React`
+
+- [**hosted link**](https://letsplaytictactoe.netlify.app/)
+
+- [**github link**](https://github.com/dikshagoyal26/react-TicTacToe)
```
Comment (gkanishk) [inline content/finalist/ttjc@finalist@dikshagoyal26/index.md:86]: This shows the title as "React App" kindly change it.

File: content/finalist/ttjc@finalist@dikshagoyal26/index.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+- [**github link**](https://github.com/dikshagoyal26/Flexible-Warehousing-Software-Backend)
+
+### News App
+
+A pseudo Frontend Application which displays news for different countries based on the category chosen by the user.
+
+- **Stack** `AngularJS`
+
+- [**github link**](https://github.com/dikshagoyal26/NewsApp)
```
Comment (gkanishk) [inline content/finalist/ttjc@finalist@dikshagoyal26/index.md:154]: Kindly remove previously made projects

File: content/finalist/ttjc@finalist@dikshagoyal26/index.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+image_link: "https://avatars3.githubusercontent.com/u/34977174?s=400&u=9831d37daead95e55fbf659f689236593f118af6&v=4"
+looking_for: "job"
+description: "JavaScript Developer| NodeJs |MongoDB |Angular | Full Stack Developer|Tech Enthusiast"
+---
+
+Namaste! I am Diksha, MEAN stack developer.
+
+## Introduction
+
+I am a senior year undergrad pursuing B.Tech degree in Electronics & Communication Engineering at Maharaja Agrasen Institute of Technology, Delhi. I am interested in software development. I am a quick learner and enthusiastic about problem-solving and exploring technology. I am a highly motivated JavaScript developer. During my work experience, I learned to handle the pressure to meet the deadline of project submission and to complete them with accuracy and efficiency. I am a keen learner and looking for more opportunities ahead.
```
Comment (gkanishk) [inline content/finalist/ttjc@finalist@dikshagoyal26/index.md:17]: Senior year? Final year will sound good?

**Comment by dikshagoyal26** [comment]
Hi Mentor,

Thank you for your feedback, now I have made the required changes, please review and accept my pull request.

Thank you.

**Comment by gkanishk** [comment]
Please provide the details here #1033 @Megha-Pathak kindly add her

**Pull request merged**